ksqldb-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java

```diff
@@ -125,6 +126,7 @@ public void before() {
   }
 
   @Test
+  @Ignore
```

Comment:
It looks like we are not cleaning up repartition topics, we should be doing this in `QueryCleanupService` but it doesn't seem to be picking them up.